File: src/java/models/Location.java
package models;

public class Location {

    private String lon, lat;
    private int ID;

    public Location(String lon, String lat, int ID) {
        this.lon = lon;
        this.lat = lat;
        this.ID = ID;
    }

    public String getLon() {
        return lon;
    }

    public String getLat() {
        return lat;
    }

    public int getID() {
        return ID;
    }
}

File: src/java/PHP Section/location.js
if (navigator.geolocation) {
    navigator.geolocation.getCurrentPosition(showPosition, showError);
} else {
    window.location = referer + "?error=location_not_supported";
}
function showPosition(position) {
    window.location = referer + "?lat=" + position.coords.latitude + "&lon=" + position.coords.longitude;
    return;
}

function showError(error) {
    switch (error.code) {
        case error.PERMISSION_DENIED:
            window.location = referer + "?error=permission_denied";
            break;
        case error.POSITION_UNAVAILABLE:
            window.location = referer + "?error=position_unavailable";
            break;
        case error.TIMEOUT:
            window.location = referer + "?error=timeout";
            break;
        case error.UNKNOWN_ERROR:
            window.location = referer + "?error=exception";
            break;
    }
}